### ISSUE TITLE
Verified status for channels

### ISSUE THREAD

**Issue by bedeho** [Joystream/joystream#2586]
Extend metadata standard for channels, importantly, this may require introducing metadata submission extrinsic that only curators can use, as only curators can manage this status. The semantics are entirely social, but they mean something to the effect of `this account is real`. There should probably be a rationale payload whenever this status is updated.

Also this constraint is probably a wise idea:
https://twitter.com/danrobinson/status/1494210892883398657

Design > 
-https://github.com/Joystream/atlas/issues/2091

**Comment by dmtrjsg** [comment]
There would be the runtime part to it..

**Comment by bedeho** [comment]
If curators use a the curator remark extrinsic, then no runtime change is needed, just a new message. The only reason this would need runtime work is _if the runtime itself needs to distinguish verified from unverified channels_. Currently, no such reason exists, but it could, e.g. if we want to introduce limits on who can mint creator tokens, as in this issue https://github.com/Joystream/joystream/issues/4151.

I think that the best way to approach this is to just start with the metaprotocol version for now, as it is the path of least resistance, and tech-debt of being wrong is quite low.

**Comment by dmtrjsg** [comment]
@bedeho yes, that's what we also settled on, and 3254 linked is the implementation for the metaprotocol. 

Runtime we concluded would be needed to enforce heuristics and disallow updating channel assets for a verified profile.

**Comment by dmtrjsg** [comment]
Update of the scope:
- [ ] Privilege to update the status of channel to "verified" would only be in the hands of Content WG Lead
- [ ] Interface to be implemented via CLI
- [ ] Mappings added to QN (as cli doesn't work with Orion yet)
- [ ] Mappings added to Orion

ℹ️ here's the issue where designs are covered:
- https://github.com/Joystream/atlas/issues/3840
